src/utils.js

```diff
@@ -109,6 +110,21 @@ const printAPIPoints = (points) => {
 \tremaining\t-\t${points.remaining}`);
 };
 
+const checkPrivateVulnerabilityReporting = async (owner, repo, token) => {
+	try {
+		const response = await request('GET /repos/{owner}/{repo}/private-vulnerability-reporting', {
+			headers: {
+				authorization: `token ${token}`,
+			},
+			owner,
+			repo,
+		});
+		return response.data.enabled;
+	} catch (error) {
+		return false;
+	}
+};
+
```

Comment:
once you fix linting errors, you will notice that these lines of code are listed as "untested" code.

it would be ideal if they were tested.

<img width="1125" height="508" alt="image" src="https://github.com/user-attachments/assets/eddb7d8a-05f3-473e-bd48-cea106307f9b" />

so if you could a test file that is similar to this one:

https://github.com/ljharb/repo-report/blob/main/test/utils/printAPIPoints.js

this will fix code coverage to not show these lines as yellow